__tests__/app.test.js

```diff
@@ -420,4 +425,66 @@ describe('GET /api/articles', () => {
         expect(body.msg).toBe("Invalid order value");
       });
     });
-  });
\ No newline at end of file
+  });
+
+  //TASK 12 (which builds on tasks 5 & 11)
+  describe('GET /api/articles', () => {
+    test('200: serves an array of articles, filtered by topic', () => {
+      return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body}) => {
+          const { articles } = body;
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
We dont really need an assertion of `toBeSortedBy` here, but instead we should have an assertion to ensure the array isnt empty so we dont get a false passing test for an empty array